app/services/messages/service.go

```diff
@@ -100,8 +104,9 @@ func (ss *Service) SanityCheckSignature(topicMessage message.Message) (bool, err
 	}
 	signedAmount := strconv.FormatInt(amount-feeAmount, 10)
 
-	wrappedAsset, err := ss.contractsService.ToWrapped(t.NativeAsset)
-	if err != nil {
+	// TODO: Discuss how this behavior will be implemented... 0 for Hashgraph?
+	wrappedAsset := ss.mappings.NativeToWrappedByNetwork[0].NativeAssets[t.NativeAsset][1]
```

Comment:
The indices here will be retrieved from the protobuf message, rather than being hardcoded